### PR TITLE
[full-ci] Init store and auth as early as possible

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Applications need e.g. capabilities in their respective ready() and
mounted() hooks. This commit ensures that they are already loaded when
applications get initialized.

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10NotificationBasic</strong> https://drone.owncloud.com/owncloud/web/19397/37/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

File: packages/web-runtime/src/container/bootstrap.ts [hunk 1]
```diff
@@ -30,6 +30,39 @@ export const requestConfiguration = async (path: string): Promise<RuntimeConfigu
   })
 }
 
+/**
+ * Prepare store with basic that are needed before any application gets announced.
+ *
```
Comment (fschade) [inline packages/web-runtime/src/container/bootstrap.ts:35]: needs update after renaming it back to "announce...", something like:
.. announce and prepare vue store ...

File: packages/web-runtime/src/container/bootstrap.ts [hunk 2]
```diff
@@ -30,6 +30,39 @@ export const requestConfiguration = async (path: string): Promise<RuntimeConfigu
...
+export const announceStore = async ({
+  vue,
+  runtimeConfiguration,
+  store
+}: {
+  vue: VueConstructor
+  runtimeConfiguration: RuntimeConfiguration
+  store: Store<any>
+}): Promise<void> => {
+  console.log('initAuth')
```
Comment (fschade) [inline packages/web-runtime/src/container/bootstrap.ts]: dev leftOver

**Review by fschade — APPROVED** [review]
two little things, else LGTM 🚀

File: packages/web-runtime/src/container/bootstrap.ts (hunk 1 above)
Comment (kulmann) [reply packages/web-runtime/src/container/bootstrap.ts:35]: 👍

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPublic</strong> https://drone.owncloud.com/owncloud/web/19399/61/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles2</strong> https://drone.owncloud.com/owncloud/web/19400/53/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by kulmann** [comment]
Acceptance tests have one stable failure for both oc10 and ocis in `tests/acceptance/features/webUIFilesSearch/search.feature:38`

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/19417/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Pull request merged**